js/components/KeyboardShortcutHUD.react.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * KeyboardShortcutHUD
+ */
+import React, { Component } from 'react';
+import Combokeys from 'combokeys'
+
+class KeyboardShortcutHUD extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { show: false };
+  }
+
+  componentDidMount() {
+    let combokeys = new Combokeys(document.documentElement);
+    combokeys.bind('?', () => { this.setState({ show: !this.state.show }); });
+  }
+
+  componentWillUnmount() {
+    let combokeys = new Combokeys(document.documentElement);
+    combokeys.unbind('?');
+  }
+
+  render() {
+    let display = this.state.show ? 'block' : 'none';
+    return (
+      <div className='keyboard-hud' style={{ display: display }}>
```

Comment:
Completely personal preference, but can we create a hidden class to use for this type of thing instead of having the styles inline?